installation/resources/installer-cr-cluster-agent.yaml.tpl

```diff
@@ -0,0 +1,81 @@
+apiVersion: "installer.kyma-project.io/v1alpha1"
```

Comment:
Please add document describing which files should be modified when a new component is added.